node/src/blockchain/test_utils.rs

```diff
@@ -56,9 +63,139 @@ pub fn make_meaningless_seed() -> Seed {
     Seed::new(&mnemonic, "passphrase")
 }
 
+pub fn make_blockchain_interface_web3(port_opt: Option<u16>) -> BlockchainInterfaceWeb3 {
+    let port = port_opt.unwrap_or_else(|| find_free_port());
+    let chain = Chain::PolyMainnet;
+    let (event_loop_handle, transport) = Http::with_max_parallel(
+        &format!("http://{}:{}", &Ipv4Addr::LOCALHOST, port),
+        REQUESTS_IN_PARALLEL,
+    )
+    .unwrap();
+
+    BlockchainInterfaceWeb3::new(transport, event_loop_handle, chain)
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
+pub struct RpcResponse<S: Serialize> {
```

Comment:
This structure doesn't resemble test code. Is it the case it doesn't take place in production at all? If it does, it doesn't belong here. 

A good way of pursuing this principle is to add `#![cfg(test)]` at the top of the file or there is an alternative to this (and I myself hate this inconsistency in the possible treatments) in the respective mod file, situated above this one in the cargo project's file system, that clusters up all the other files in the directory (you may know very well, as these are those lines starting with the `mod` keyword), which is where you can simply localise your particular file (test_utils.rs) and superscribe it by `[cfg(test)]`.

No matter what solution you chose this will always play in your favour because everything which would be accidentally declared inside but used also in the production code would inevitably direct to a compilation error.